src/main/resources/static/api.yaml

```diff
@@ -730,6 +730,9 @@ components:
             $ref: '#/components/schemas/ConsentApproveRequest'
           xml:
             wrapped: true
+        callBackUrl:
+          type: string
+          example: 'http://tmh-hiu.com/'
```

Comment:
```suggestion
          example: 'https://tmh-hiu.com/notify'
```

Reply:
It should not be part of approve request no? ConsentRequest only should have it no?

Reply:
Can you remove the callback url from consent approve request? and added it to the consentrequest?